### PR TITLE
Template changes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- updates templates that clipboard is loaded next to changelist
- adds change and delete link ti same td as move to link

Before:
screen shot 2015-09-16 at 14 21 50

After:

screen shot 2015-09-17 at 09 48 19

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

File: filer/templates/admin/filer/folder/directory_table.html [hunk 1]
```diff
@@ -1,96 +1,94 @@
...
+                    <td>
+                        <div><strong><a href="{{ subfolder.get_admin_directory_listing_url_path }}{% if is_popup %}?_popup=1{% if select_folder %}&select_folder=1{% endif %}{% endif %}" title="{% blocktrans with subfolder.name as item_label %}Change '{{ item_label }}' folder details{% endblocktrans %}">{{ subfolder.name }}</a></strong></div>
+                    </td>
+                    <td>
+                        {% if not subfolder.is_root %}<span class="tiny"> ({% blocktrans count subfolder.children_count as counter %}{{ counter }} folder{% plural %}{{ counter }} folders{% endblocktrans %}, {% blocktrans count subfolder.file_count as counter %}{{ counter }} file{% plural %}{{ counter }} files{% endblocktrans %})</span>{% endif %}
+                    </td>
+                    <td>
+                        {% trans "Owner" %}: {{ subfolder.owner|default:"n/a" }}
+                    </td>
+                    <td style="text-align: right;">
```
Comment (vxsx) [inline filer/templates/admin/filer/folder/directory_table.html]: please don't do inline styles (at least for new changes)

File: filer/templates/admin/filer/folder/directory_table.html [hunk 2]
```diff
@@ -1,96 +1,94 @@
...
+                        {% if enable_permissions %}<div>{% trans "Permissions" %}: {% if file.is_public %}{% trans "disabled" %}{% else %}{% trans "enabled" %}{% endif %}</div>{% endif %}
+                        {# {% if file.has_all_mandatory_data %}<div><img alt="0" src="{% admin_icon_base %}img/admin/icon-yes.gif" /> {% trans "has all mandatory metadata" %}</div>{% else %}<div><img alt="1" src="{% admin_icon_base %}img/admin/icon-no.gif"/> {% trans "missing metadata!" %}</div>{% endif %} #}
+                    </td>
+                    <td>
+                        <span class="tiny"> ({{ file.size|filesize:"auto1000long" }}{% ifequal file.file_type "Image" %}, {{ file.width }}x{{ file.height }} px{% endifequal %})</span>
+                    </td>
+                    <td>
+                        {% trans "Owner" %}: {{ file.owner|default:"n/a" }}
+                    </td>
+                    <td class="moveto" style="text-align: right;">
```
Comment (vxsx) [inline filer/templates/admin/filer/folder/directory_table.html]: same here

File: filer/static/filer/css/admin_style.css [hunk 3]
```diff
@@ -61,13 +61,31 @@ table thead th.thumbHeader {
...
+.filebrowser #changelist .moveto {
+	text-align: right;
+	min-width: 180px
+}
+
+.filebrowser #changelist .moveto a {
+	display: inline-block; margin-right: 10px; margin-top: 10px;
+}
+.filebrowser #changelist .moveto button {
+float: right;
```
Comment (vxsx) [inline filer/static/filer/css/admin_style.css]: intentation

**Comment by vxsx** [comment]
codewise LGTM

**Comment by stefanfoulis** [comment]
how does it look with the default django admin styling?

**Comment by lory87** [comment]
@stefanfoulis 
screen shot 2015-09-17 at 16 23 20

**Comment by lory87** [comment]
@stefanfoulis without admin styles
screen shot 2015-09-17 at 17 11 23

**Comment by stefanfoulis** [comment]
the default admin version (without any of our special styling) looks a bit broken now (e.g no padding around "clipboard", etc).

**Comment by lory87** [comment]
screen shot 2015-09-18 at 12 04 42
screen shot 2015-09-18 at 11 57 06
screen shot 2015-09-18 at 11 56 47

**Comment by vxsx** [comment]
codewise is ok

**Comment by stefanfoulis** [comment]
@lory87 cool, thanks. I'm really looking forward to the new styled version.

@yakky @mkoistinen what do you guys think? Is the default admin-styling version good enough like that?

**Pull request merged**